.github/workflows/ExoBLAS.yml

```diff
@@ -0,0 +1,13 @@
+name: ExoBLAS
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  call_test_ExoBLAS:
+    # point to reusable-workflow branch for testing before merging
+    uses: exo-lang/ExoBLAS/.github/workflows/test_ExoBLAS.yml@reusable-workflow
+    with:
+      exo-ref: ${{ github.event.pull_request.head.ref }}"
```

Comment:
How does `github.event.pull_request.head.ref` gonna work on main?

Reply:
Yeah I need to check what triggered the action and decide the ref based on it. But I was trying to get it to work on the pull request ref for starters.

Reply:
I see that makes sense, lmk when you want to merge this